From 8a4cad5e2f2cbb6a34bdc6e877fe499502b8c4c8 Mon Sep 17 00:00:00 2001
From: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date: Fri, 23 Dec 2016 18:12:29 +0100
Subject: [PATCH] linuxptp: Use cross cpp in incdefs

Use cross cpp incdefs.sh shell script since we are doing cross compiling
we need to ensure we use correct setttings from toolchain

Upstream-Status: Inappropriate [OE-Specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---

 makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/incdefs.sh
+++ b/incdefs.sh
@@ -27,7 +27,7 @@ user_flags()
 	printf " -D_GNU_SOURCE"
 
 	# Get list of directories searched for header files.
-	dirs=$(echo "" | ${CROSS_COMPILE}cpp -Wp,-v 2>&1 >/dev/null | grep ^" /")
+	dirs=$(${CPP} -Wp,-v -xc /dev/null 2>&1 >/dev/null | grep ^" /")
 
 	# Look for clock_adjtime().
 	for d in $dirs; do
